addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -150,6 +150,29 @@ def _compute_plan_id(self):
     def _compute_plan_date(self):
         self.plan_date = False
 
+    @api.depends('plan_date', 'plan_id')
+    def _compute_plan_summary(self):
+        template_summaries_list_by_plan = {}
+        for template in self.plan_id.template_ids:
+            template_summary = f": {template.summary}" if template.summary else ''
+            template_summaries_list_by_plan.setdefault(template.plan_id, []).append(
+                (template, template.activity_type_id.name, template_summary))
```

Comment:
Would say the previous code was a bit more simple.

As we will need to fetch all the "summary" field of all the template_ids anyway, even if we have several times the same templates, the cache should do its work.